apprise/plugins/NotifyDBus.py

```diff
@@ -77,19 +83,26 @@
     NOTIFY_DBUS_SUPPORT_ENABLED = (
         LOOP_GLIB is not None or LOOP_QT is not None)
 
+    # ImportError: When using gi.repository you must not import static modules
+    # like "gobject". Please change all occurrences of "import gobject" to
+    # "from gi.repository import GObject".
+    # See: https://bugzilla.gnome.org/show_bug.cgi?id=709183
+    if "gobject" in sys.modules:  # pragma: no cover
+        del sys.modules["gobject"]
+
     try:
         # The following is required for Image/Icon loading only
         import gi
         gi.require_version('GdkPixbuf', '2.0')
         from gi.repository import GdkPixbuf
         NOTIFY_DBUS_IMAGE_SUPPORT = True
 
-    except (ImportError, ValueError, AttributeError):
+    except (ImportError, ValueError, AttributeError) as ex:
```

Comment:
My only comment on this thread would be to drop all the captures of the logging failures here and return to `pass` for these 3 files (DBus, Gnome, and MacOSX).  Import errors should silently fail (a global flag gets set to False for reasons later).  But we don't want Operating systems like Windows to see these errors on every single call to Apprise. :slightly_smiling_face:

Reply:
Thanks. I've fixed it.